docs/layout-of-source-files.rst

```diff
@@ -79,7 +79,7 @@ be sure that your code compiles the way you intended. The exact version of the
 compiler is not fixed, so that bugfix releases are still possible.
 
 It is possible to specify more complex rules for the compiler version,
-these follow the same syntax used by `npm <https://docs.npmjs.com/cli/v6/using-npm/semver>`_.
+these follow the same syntax used by `npm <https://nodejs.dev/learn/semantic-versioning-using-npm>`_.
```

Comment:
I think the first link is better, to be honest.

Reply:
for some reason that link did not work for me when I was trying to read click on it.

Reply:
ok, i must have used an older version at readthedocs maybe. will drop this commit.